docs/gettsim_objects/input_variables.md

```diff
@@ -516,3 +516,9 @@ Type: float
 Tax Bracket (1 to 5) for withholding tax
 
 Type: int
+
+## `anz_kinder_u_25`
+
+Amount of children below tha age of 25
```

Comment:
```suggestion
Amount of own children below the age of 25 (incl. not in hh)
```

I assume not in hh count as, well? Otherwise, we won't need a new input variable.